packages/palette/src/elements/Marquee/Marquee.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { useState, useEffect, useRef } from "react"
+import styled, { keyframes } from "styled-components"
+import { variant } from "styled-system"
+import { TextVariant } from "@artsy/palette-tokens/dist/typography/types"
+import { useThemeConfig } from "../../Theme"
+import { Flex } from "../Flex"
+import { Box, BoxProps } from "../Box"
+import { Text } from "../Text"
+
+const VARIANTS = {
+  defaultLight: {
+    backgroundColor: "black10",
+    color: "black100",
+  },
+  defaultDark: {
+    backgroundColor: "black100",
+    color: "white100",
+  },
+  brand: {
+    backgroundColor: "brand",
+    color: "white100",
+  },
+}
+
+const move = (props) => keyframes`
+  from {
+    transform: translateX(0);
+  }
+  to {
+    transform: translateX(-${props.offset}px);
+  }
+`
+
+const Container = styled(Box)`
+  ${variant({ variants: VARIANTS })}
+  width: 100%;
+  overflow-x: hidden;
+`
+const StyledText = styled(Text)`
+  ${variant({ variants: VARIANTS })}
+  white-space: nowrap;
+`
+
+const Inner = styled.div<{ speed: string; offset: number }>`
+  animation: ${move} ${(props) => props.speed} infinite linear;
+  cursor: default;
+  user-select: none;
+  white-space: nowrap;
+`
+
+const Item = styled.div`
+  display: inline-block;
+`
+
+export interface MarqueeProps extends BoxProps {
+  variant?: keyof typeof VARIANTS
+  speed?: string
+  color?: string
+  marqueeText: string
+  noDivider?: boolean
+}
+
+export const Marquee: React.FC<MarqueeProps> = ({
+  marqueeText,
+  noDivider = false,
```

Comment:
**nitpick [non-blocking]:** prefer truthy flag names

I have a slight preference for flag names to be truthy so that you don't have to have double negatives and break your brain haha. So what if this was `divider` instead and the default was `true` and then in the actual code we'd have:

```
{divider &&
  <StyledText>...
}
```

*format via [Conventional Comments](https://conventionalcomments.org/)*

Reply:
+

Reply:
I had that originally but changed it so that you could pass `noDivider` instead of `divider = false` but happy to change it back!